scripts/schema-check-files.ts

```diff
@@ -15,9 +15,6 @@ async function main() {
 
   // throws (non-zero exit) on any problem
   validateScenarios(entries);
-  console.log(
-    `✔ Validated ${names.length} data file(s) from ${dir} against schema.`,
-  );
```

Comment:
I think this one is actually a feature?

It gives useful logging info when the action is run, e.g.:
https://github.com/RMI/pbtar/actions/runs/17467716232/job/49607759465

<img width="699" height="450" alt="Screenshot 2025-09-22 at 17 19 12" src="https://github.com/user-attachments/assets/dd3ee78c-1391-40d6-bd70-2eb3a78306e1" />